docs/user/planner/signup_methods.rst

```diff
@@ -0,0 +1,57 @@
+Signup methods
+==============
+
+To cover different use cases, ephios offers different signup methods. In the following, the provided logon procedures are explained, but additional signup methods can also be integrated as a plugin.
```

Comment:
I think this misses the point of explaining what signup methods actually are. "Logon" is something I'd connect to my login credentials. I think we should motivate why we provide different signup methods.

The next paragraph should maybe come first.